lib/alephant/preview/server.rb

```diff
@@ -27,6 +27,10 @@ class Server < Sinatra::Base
 
       BASE_LOCATION = "#{(ENV['BASE_LOCATION'] || Dir.pwd)}/components"
 
+      before do
+        response["Access-Control-Allow-Origin"] = "*"
```

Comment:
We are making a JS post request in our browser for the components. I get a cross access error if this isn't present.